source/sfp/1_water/sfp_114_nl_waterstaalname_plas/01_afhankelijkheden.Rmd

```diff
@@ -1,17 +1,47 @@
 # Afhankelijkheden
 
-Procedures waarnaar in deze procedure verwezen wordt:
-
--   SOP-005: Richtlijnen voor conserveren van watermonsters, chemische variabelen en praktische handelingen op terrein
-
--   SVP-015: Bioveiligheidsmaatregelen (in opmaak)
-
--   SVP-112: Veiligheid in en rond water (in opmaak)
-
-Procedures die verwijzen naar deze procedure:
-
--   SPP-116: Abiotische staalname stilstaande wateren
-
--   SPP-117: Abiotische staalname stromende wateren
-
--   SVP-115: Veldmeting abiotiek oppervlaktewater met behulp van WTW Multi 3430 veldmeter
```

Comment:
SVP-115, SPP-116 en SPP-117 heb ik ook nog staan en moeten van mijn team online worden gezet (voor eind augustus), dus ik zou dit laten staan?

Reply:
Ik zou het niet doen. Pas wanneer je SVP-115 omzet, kan je verwijzen naar sfp-114-nl van zodra dit gepubliceerd is.